django/contrib/admin/sites.py

```diff
@@ -65,10 +65,14 @@ def __init__(self, name='admin'):
         self._global_actions = self._actions.copy()
         all_sites.add(self)
 
-    def check(self):
+    def check(self, app_configs):
+        if app_configs is not None:
+            app_configs = set(app_configs)  # Speed up lookups below
+
         errors = []
         for admin_obj in six.itervalues(self._registry):
-            errors.extend(admin_obj.check())
+            if app_configs is None or admin_obj.model._meta.app_config in app_configs:
```

Comment:
Ah okay, I wasn't following the [docs](https://docs.djangoproject.com/en/1.10/topics/checks/):

> The check function must accept an app_configs argument; this argument is the list of applications that should be inspected. If None, the check must be run on all installed apps in the project.